### ISSUE TITLE
Accessibilty

### ISSUE THREAD

**Issue by cpsievert** [rstudio/bootstraplib#80]
Shiny is now including CSS from https://github.com/paypal/bootstrap-accessibility-plugin by default for improved BS3 accessibility. For this reason, perhaps `bs_theme_new(3)` should be layering in its Sass by default https://github.com/paypal/bootstrap-accessibility-plugin/tree/main/src/sass

We'll have to also assess BS4 accessibility...
